docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -23,8 +23,8 @@ The Grafana 7.0 stable release is scheduled for the 18th of May. In the meantime
 The main highlights are:
 
 - [**New Panel Editor** Redesign based on community feedback.]({{< relref "#new-panel-editor" >}})
-- [**Explore:** New tracing UI and support for visualizing Jaeger and Zipkin traces.]({{< relref "#new-tracing-ui" >}})
-- [**Enterprise** Usage Analytics and Presence.]({{< relref "#grafana-enterprise" >}})
+- [**Explore** New tracing UI and support for visualizing Jaeger and Zipkin traces.]({{< relref "#new-tracing-ui" >}})
+- [**Enterprise** Usage Analytics, Presence and Auth Improvements.]({{< relref "#grafana-enterprise" >}})
```

Comment:
```suggestion
- [**Enterprise** Usage Insights, Presence Indicator and Auth Improvements.]({{< relref "#grafana-enterprise" >}})
```
It would be nice to update this in the subheader as well (https://github.com/grafana/grafana/blob/master/docs/sources/guides/whats-new-in-v7-0.md#usage-analytics-and-presence)

Reply:
And apply the serial comma after "Presence indicator".

Reply:
Good points, thanks!